framework/helpers/BaseHtml.php

```diff
@@ -929,7 +929,8 @@ public static function checkboxList($name, $selection = null, $items = [], $opti
      * @param array $options options (name => config) for the radio button list container tag.
      * The following options are specially handled:
      *
-     * - tag: string, the tag name of the container element.
+     * - tag: string|false, the tag name of the container element. Disable the container of the
```

Comment:
- tag: string|false, string is the tag name of the container element. False to render radio buttons without container.